projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -192,8 +192,17 @@ function ( $viewer ) use ( $args ) {
 		foreach ( array_keys( $this->response_format ) as $key ) {
 			switch ( $key ) {
 				case 'found':
-					$user_count     = (int) $user_query->get_total();
-					$viewer_count   = $include_viewers ? count( $viewers ) : 0;
+					$user_count = (int) $user_query->get_total();
+
+					$viewer_count = 0;
+					if ( $include_viewers ) {
+						if ( empty( $args['search'] ) ) {
+							$viewer_count = (int) get_count_private_blog_users( $blog_id );
```

Comment:
Since `get_count_private_blog_users` is a WordPress.com function, should we have a check for wpcom before we try to call it?

Reply:
@jeherve  `$include_viewers` should have that check built-in, [see this](https://github.com/Automattic/jetpack/pull/29068/files#diff-8663281e2c4cca6538a8b94162c2a5b34f920a629dbfc068fc83241cf2a70e34L167). Thanks for the double-check!

Reply:
I was thinking about the same thing but $include_viewers is only true if we are on wpcom. Maybe it wouldn't hurt to add it there again, for clarity.